e2e/__tests__/__snapshots__/beforeEachAbortsTests.test.ts.snap

```diff
@@ -0,0 +1,435 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I also considered only using `stdout`’s `console.log`s to determine what has been run and what hasn’t, but I considered it was important to check the failures that are actually displayed.

An example of this alternative approach can be found [here](https://github.com/facebook/jest/pull/6673/files). It looks very nice and a lot more concise, but I’m not sure it would be the optimal approach for this PR itself.